### ISSUE TITLE
Nice to have: consistent use of colors in log and access attempt entries per app

### ISSUE THREAD

**Issue by Gerolianis** [M66B/NetGuard#411]
Just a suggestion to improve readability: I would be happy to see right column of log entries displayed using the same colors as internet access attempt entries (at app level) eg. orange for blocked and bluegreen for unblocked. From the symbols (as they are now) it's possible to deduct what happened, but coloring would make browsing logs somewhat easier.

**Comment by M66B** [comment]
I think this would not look nice and uneasy.
